server/rooms.ts

```diff
@@ -111,6 +111,7 @@ export abstract class BasicRoom {
 	gtsga: GTSGiveaway | null;
 	toursEnabled: '%' | boolean;
 	tourAnnouncements: boolean;
+	tierDisplay: string;
```

Comment:
```suggestion
	dataCommandTierDisplay: string;
```

I've started preferring longer names because I'd rather not have people see this definition and not know what it means.

Reply:
(A jsdoc comment would work too, but considering this variable is rarely ever used, I'd rather just rename the variable.)

Reply:
I do not know what you mean by jsdoc. You keep using it when you comment on my pull requests and I've never known what that means.

Reply:
/** foo */
https://jsdoc.app/about-getting-started.html